server/odc-service/src/main/java/com/oceanbase/odc/service/iam/UserBatchImportPreviewer.java

```diff
@@ -92,7 +91,7 @@ public UserPreviewBatchImportResp preview(MultipartFile file) throws IOException
             BatchImportUser batchImportUser;
             try {
                 batchImportUser = createBatchImportUser(map, accountNames, roleName2RoleMap);
-            } catch (VerifyException e) {
+            } catch (HttpException e) {
```

Comment:
we may diff the error code of the httpException to make sure that the exception is thrown by password validation~

Reply:
the field errorMessage is not design for password validation, but for whole user validation.
I change from VerifyException to HttpException due all HttpException contains i18n error message.